### PR TITLE
fix name in barlouie

### PR DESCRIPTION
**Pull request by matkoniecz**

### CODE REVIEW

File: locations/spiders/barlouie.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 class BarlouieSpider(SitemapSpider):
     name = "barlouie"
     allowed_domains = ["www.barlouie.com"]
-    item_attributes = {"brand": "Barlouie", "brand_wikidata": "Q16935493"}
+    item_attributes = {"brand": "Bar Louie", "brand_wikidata": "Q16935493"}
```
Comment (Cj-Malone) [inline locations/spiders/barlouie.py:11]: NSI has this as `brand=Bar Louie Restaurants`

https://nsi.guide/index.html?t=brands&k=amenity&v=bar&tt=q16935493

Comment (matkoniecz) [reply locations/spiders/barlouie.py:11]: https://www.openstreetmap.org/search?query=Bar+Louie

https://www.google.com/maps/search/Bar+Louie/

https://commons.wikimedia.org/w/index.php?search=Bar%20Louie

https://commons.wikimedia.org/w/index.php?search=Bar+Louie+Restaurants

https://www.google.com/maps/search/Bar+Louie+Restaurants/

https://www.openstreetmap.org/search?query=Bar%20Louie%20Restaurants

Comment (Cj-Malone) [reply locations/spiders/barlouie.py:11]: If you think NSI is wrong, update NSI. OSM will drift towards NSI, not ATP.

Comment (matkoniecz) [reply locations/spiders/barlouie.py:11]: well, I can only request NSI fixes, I cannot just update it

https://github.com/osmlab/name-suggestion-index/pull/10126

**Pull request merged**